web/src/test/java/org/openmrs/web/context/AltAuthSchemeContextTest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.web.context;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.api.context.Authenticated;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.context.Credentials;
+import org.openmrs.api.context.TestUsernameCredentials;
+import org.openmrs.web.test.BaseWebContextSensitiveTest;
+
+/**
+ * A Context loaded with a test Spring config that overrides the default auth scheme.
+ * 
+ * @see Context
+ */
+/*
+ * TODO Enable @ContextConfiguration on subclasses of BaseContextSensitiveTest
+ * 
+ * @see https://issues.openmrs.org/browse/TRUNK-381?focusedCommentId=254292&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-254292
+ */
+//@ContextConfiguration(locations = { "classpath*:AltAuthSchemeTestingApplicationContext.xml" })
```

Comment:
Since our practice is not to have commented out code, would you instead create tickets for things you want addressed later?

Reply:
The same goes for TODO